unison-share-api/src/Unison/Server/Endpoints/FuzzyFind.hs

```diff
@@ -52,7 +53,7 @@ import Unison.Util.Pretty (Width)
 
 type FuzzyFindAPI =
   "find" :> QueryParam "rootBranch" SBH.ShortBranchHash
-    :> QueryParam "relativeTo" HashQualifiedName
+    :> QueryParam "relativeTo" Path.Path
```

Comment:
I updated some of these API types to be more strongly typed so we don't have to do parsing in the handlers.